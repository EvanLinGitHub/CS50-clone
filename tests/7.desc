get_int - rejects non-ints
