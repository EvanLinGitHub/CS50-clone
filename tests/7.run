./get_int < 7.in
