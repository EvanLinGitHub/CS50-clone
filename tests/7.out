Int: Int: Int: 50
